File: Source/Core/Utility/UIBarItem+Utility.m
//
//  UIBarItem+Utility.m
//  Eunomia
//
//  Created by Ian on 10/20/14.
//  Copyright (c) 2014 Eunomia. All rights reserved.
//

#import "UIBarItem+Utility.h"

@implementation UIBarItem (Eunomia_Utility)

- (UIView *)view
{
    UIView *result;
    
    id object = [self valueForKey:@"view"]; // HACK: >:#
    if ( [object isKindOfClass:[UIView class] ] )
    {
        result = object;
    }
    
    return result;
}

@end

File: Source/Core/UtilityExtensions/Views/UIBarItem+Utility.swift
//
//  UIBarItem+Utility.swift
//  Pods
//
//  Created by Ian Grossberg on 11/26/15.
//
//

import UIKit

extension UIBarItem {
    
    public var view : UIView? {
        return self.value(forKey: "view") as? UIView
    }
}
